.circleci/config.yml

```diff
@@ -25,8 +25,12 @@ jobs:
     executor: linux
     steps:
       - checkout
-      - restore_cache:
-          key: deps-{{ .Branch }}--<< parameters.node-version >>--{{ checksum "package.json" }}
+      - run:
```

Comment:
I would back this change out. I'll be making a document on how to bust the cache on PRs here: https://github.com/nasa/openmct/issues/4576

Reply:
@unlikelyzero I'm not sure why this change is here. Maybe it was an accidental merge leftover. It doesn't seem to have anything to do with linting and I have no recollection of it.

Reply:
Hmm, yeah, looks like those lines were from Nikhil's commit here: https://github.com/nasa/openmct/commit/b2f4459093a54bb2e1011d9e4188618871e83e07

A merge probably snuck it in unintentionally. I'll just copy the whole file over with whatever is on master.